packages/babel-plugin-minify-dead-code-elimination/src/index.js

```diff
@@ -895,4 +895,10 @@ module.exports = ({ types: t, traverse }) => {
       };
     }
   }
+
+  // things that are hoisted
+  function canExistAfterCompletion(path) {
```

Comment:
That test is already covered in `should be fine with fun decl after return` if that's what you meant.